models/userDao.js

```diff
@@ -0,0 +1,94 @@
+const appDataSource = require('./dataSource');
+
+const getUserIdByKakaoId = async(kakaoId) => {
+  return await appDataSource.query(`
+    SELECT
+      id AS userId
+    FROM users
+    WHERE kakao_id = ?;`, [ kakaoId ]
+  );
+}
+
+const storeKakaoUserInfo = async(kakaoUserInfo) => {
+  const kakaoId = kakaoUserInfo.id;
+  const nickname = kakaoUserInfo['kakao_account'].profile.nickname;
+  const profileImageUrl = kakaoUserInfo['kakao_account'].profile['profile_image_url'];
+  const email = kakaoUserInfo['kakao_account'].email;
+  const birthday = kakaoUserInfo['kakao_account'].birthday;
+  let gender = kakaoUserInfo['kakao_account'].gender;
+  let genderId = 0;
+
+  if (gender = 'male') {
+    genderId = 1;
+  } else if (gender = 'female') {
+    genderId = 2;
+  } else {
+    genderId = 3;
+  }
```

Comment:
만약 객체 (enum)를 통해 매핑을 하게 된다면 다음과 같이 사용할 수 있을 것 같습니다.

```suggestion
  let genderId = 0;
  const GENDER = Object.freeze({
    male : 1,
    female: 2,
    other: 3
  })
  genderId = GENDER[gender]
```

Reply:
자료를 봐도 이해가 잘 안갔는데 이렇게 사용할 수 있겠네요! 적용해서 수정하겠습니다. 감사합니다~